x-pack/test/reporting/configs/chromium_functional.js

```diff
@@ -4,25 +4,31 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { getFunctionalConfig } from './functional';
-
 export default async function ({ readConfigFile }) {
-
-  const functionalConfig = await getFunctionalConfig({ readConfigFile });
+  const functionalConfig = await readConfigFile(require.resolve('../../functional/config.js'));
 
   return {
-    ...functionalConfig,
+    services: functionalConfig.get('services'),
+    pageObjects: functionalConfig.get('pageObjects'),
+    servers: functionalConfig.get('servers'),
+    apps: functionalConfig.get('apps'),
+    screenshots: functionalConfig.get('screenshots'),
```

Comment:
Why these 5 lines vs the destructuring that was here?

Reply:
`functionalConfig` is no longer a plain object: it used to be, and that was a side-effect on how the `import` used to be. Now we're using `readConfigFile` instead of relying on a custom function.

As it is a real config object now, if we try to spread the properties, they'll all be `undefined`. The properties need to be accessed with `.get()` for config objects.

Reply:
Ok, I usually do a `.getAll()` here, but lgtm